tabsie/TABSIEHelpers.R

```diff
@@ -4,28 +4,25 @@ toggleMaster <- function(toggleOn, toggleAll){
   if(length(toggleOn) > 0)sapply(toggleOn, function(x){shinyjs::show(id = x, anim= TRUE) })
   if(length(toggleOff) > 0)sapply(toggleOff, function(x){shinyjs::hide(id = x, anim= TRUE) })
 }
+
 htmlLabelInfo <- function(label = "This is a label.", title="PopUp Title", content="PopUpContext"){
   html1 = "<a herf= '#' class='btn btn-primary btn-xs' data-toggle='popover' data-placement= 'auto bottom' title='"
   html2 = "' data-content='"
   html3 = "'>?</a>"
   return (HTML(paste(label, html1,title, html2, content, html3)))
 }
+
 getpData<- function(filterOption, filterDic, dataList){
   #this function is another of those nice little functions that does something easy, but 
   #that we don't want to have to program elsewhere.
   line = which(filterDic == filterOption) #which filter is activated?
   
-  if(!any(line)){
+  if(!any(line)||line > length(dataList)){
     #Well! That's not supposed to happen! If we don't return things break, so we'll return the
```

Comment:
As far as I can tell, these two conditions have the same effect, so I combined them in one or condition.